### PR TITLE
adjust React import statement in `index.d.ts`

### PR DESCRIPTION
**Pull request by dsifford**
fixes: https://github.com/davidtheclark/focus-trap-react/pull/21#discussion_r157601575

### CODE REVIEW

**Comment by TedDriggs** [comment]
Thanks @dsifford!

**Comment by davidtheclark** [comment]
Published in 3.1.1

File: index.d.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ declare module "focus-trap-react" {
 
   // Extend AllHTMLAttributes to provide completions on DOM properties wherever
   // React does.
-  export interface Props extends React.AllHTMLAttributes<any> {
+  interface Props extends React.AllHTMLAttributes<any> {
```
Comment (TedDriggs) [inline index.d.ts]: @dsifford somehow I didn't notice last time that you removed the export declaration here. Is it possible to restore that at some point? When declaring an HOC or the like, it's useful to be able to name the props interface.

Comment (dsifford) [reply index.d.ts]: Ah, yep good point... It'll require another rework though. Can't be done in the form that exists currently. 

I'll submit another PR for it in a couple mins.

**Pull request merged**